CHANGES.md

```diff
@@ -15,7 +15,15 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- Add a Magpie Webhook to create the Magpie resources corresponding to the STAC-API path elements when a `STAC-API` 
+  `POST /collections/{collection_id}` or `POST /collections/{collection_id}/items/{item_id}` request is accomplished.
```

Comment:
Do we also need a hook when a collection or item is deleted? (this is the sort of thing I mean: https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features/extensions/transaction/#tag/Transaction/operation/deleteFeature)

Reply:
I dont think it would be something to do atm. The `STAC` items are generated from files found on `thredds`. There shouln't be a request to delete an item sent to `STAC` if the file is still on `thredds`. AFAIK there is no mechanism in place to notify `STAC` when an item or directory is deleted on `thredds` / if it's something we should implement in the futur. What do you think? @fmigneault

Reply:
I could be done in spirit of cleanup in Magpie, but if the resource is actually deleted in STAC, only a 404 would be returned instead of 403. So not high priority IMO.